File: ios/RecordWave/AudioMeterPackage.m
// AudioMeterPackage.m
#import "AudioMeterPackage.h"
#import <React/RCTBridge.h>

@implementation AudioMeterPackage
RCT_EXPORT_MODULE();

- (NSArray<id<RCTBridgeModule>> *)createNativeModules:(RCTBridge *)reactContext {
  return @[[[AudioMeterModule alloc] init]];
}

- (NSArray<RCTViewManager *> *)createViewManagers:(RCTBridge *)reactContext {
  return @[];
}

@end

File: ios/RecordWave/AudioMeterPackage.swift
// AudioMeterPackage.swift
import React

@objc(AudioMeterPackage)
class AudioMeterPackage: NSObject, RCTBridgeModule {
    static func moduleName() -> String! {
        return "AudioMeterPackage"
    }
    
    static func requiresMainQueueSetup() -> Bool {
        return true
    }
    
    func createNativeModules(_ reactContext: RCTBridge!) -> [RCTBridgeModule] {
        return [AudioMeterModule()]
    }
    
    func createViewManagers(_ reactContext: RCTBridge!) -> [RCTViewManager] {
        return []
    }
}